docs/src/use/getting-started.md

```diff
@@ -58,43 +58,19 @@ yarn run eslint yourfile.js
 
 **Note:** If you are coming from a version before 9.0.0 please see the [migration guide](configure/migration-guide).
 
-After running `npm init @eslint/config`, you'll have an `eslint.config.js` (or `eslint.config.mjs`) file in your directory. In it, you'll see some rules configured like this:
+When you run `npm init @eslint/config` ,terminal would provide you different options to create a config file `eslint.config.js` (or `eslint.config.mjs`) as per your need.
 
-```js
-// eslint.config.js
-export default [
-    {
-        rules: {
-            "no-unused-vars": "error",
-            "no-undef": "error"
-        }
-    }
-];
-```
-
-The names `"no-unused-vars"` and `"no-undef"` are the names of [rules](../rules) in ESLint. The first value is the error level of the rule and can be one of these values:
```

Comment:
I think we should keep this content as a next step, showing how to individually configure rules after the config file is generated for you.

Reply:
I think it's good to keep it in configuration section just below we explained the `pluginJs.configs.recommended` we can write something like:
>Then we can configure rules as following:

and then the example configuration with rules, then the para explaining rules in example?